lib/mail_automation/client.rb

```diff
@@ -17,13 +17,11 @@ class Client < Common::Client::Base
     #   claim_id: 1234
     #   file_number: 1234
     #   form526: {}
-    #   form526_uploads: []
     # })
     def initialize(params)
       @claim_id = params[:claim_id]
       @file_number = params[:file_number]
       @form526 = params[:form526]
-      @form526_uploads = params[:form526_uploads]
```

Comment:
Removing `form526_uploads` from the input params has no impact because they were never passed in [by 526 app code](https://github.com/department-of-veterans-affairs/vets-api/blob/3d5b15e/app/models/concerns/form526_claim_fast_tracking_concern.rb#L315-L319) in the first place.